jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildImageMojo.java

```diff
@@ -243,7 +279,11 @@ private ImageReference getImageReference() throws MojoFailureException {
     }
   }
 
-  private <T extends Exception> void provideSuggestionForException(
+  /**
+   * Wraps an exception in a {@link MojoExecutionException} and provides a suggestion on how to fix
+   * the error.
+   */
+  private <T extends Throwable> void provideSuggestionForException(
```

Comment:
Does this name still fit this method? Could we make it something along the lines of `throwMojoExceptionWithHelpMessage(...)`, or just anything that indicates this method is throwing something deliberately?